src/bench/checkblock.cpp

```diff
@@ -8,11 +8,36 @@
 #include <validation.h>
 #include <streams.h>
 #include <consensus/validation.h>
+#include <rpc/blockchain.h>
+
+#include <univalue.h>
 
 namespace block_bench {
 #include <bench/data/block413567.raw.h>
 } // namespace block_bench
 
+static void SerializeJSONBlockTest(benchmark::State& state) {
+    CDataStream stream((const char*)block_bench::block413567,
+            (const char*)block_bench::block413567 + sizeof(block_bench::block413567),
+            SER_NETWORK, PROTOCOL_VERSION);
+    char a = '\0';
+    stream.write(&a, 1); // Prevent compaction
+
+    CBlock block;
+    stream >> block;
+
+    CBlockIndex blockindex;
+    const uint256 blockHash = block.GetHash();
+    blockindex.phashBlock = &blockHash;
+    blockindex.nBits = 403014710;
+
+    while (state.KeepRunning()) {
+        (void)blockToJSON(block, &blockindex, &blockindex, /*verbose*/ true);
+    }
+}
+
+BENCHMARK(SerializeJSONBlockTest, 10);
```

Comment:
These are usually together in the end.

Reply:
`10` seems to be a lot, on my system (2,9 GHz Intel Core i9):
```
time ./src/bench/bench_bitcoin -filter=SerializeJSONBlockTest
# Benchmark, evals, iterations, total, min, max, median
SerializeJSONBlockTest, 5, 10, 10.0509, 0.197745, 0.20436, 0.200661
./src/bench/bench_bitcoin -filter=SerializeJSONBlockTest  9.54s user 1.08s system 90% cpu 11.755 total
```
I think @MarcoFalke said that each should take around 4 seconds?

Reply:
On `i5-8250U` 10 looks fine?
```
$ time ./src/bench/bench_bitcoin -filter=BlockToJsonVerbose
# Benchmark, evals, iterations, total, min, max, median
BlockToJsonVerbose, 5, 10, 5.64343, 0.104094, 0.118915, 0.112168

real	0m6.397s
user	0m6.183s
sys	0m0.202s
```

Reply:
> I think @MarcoFalke said that each should take around 4 seconds?

Yeah, they should take as much as all other tests take on average, which is 4 seconds on my machine. Though, this will vary based on what hardware you use.

Reply:
How much I should set? I set 10 because it's give nearly 1s which based on this comment: https://github.com/bitcoin/bitcoin/blob/332c6134bb15384e5b91c631e821fe52a591d3bc/src/bench/bench.h#L137